src/core.mjs

```diff
@@ -0,0 +1,279 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { inspect } from 'node:util'
+import { spawn } from 'node:child_process'
+import { chalk, psTree, which } from './goods.mjs'
+import { getCtx, runInCtx, setRootCtx } from './als.mjs'
+import { printStd, printCmd } from './print.mjs'
+import { formatCmd, quote } from './guards.mjs'
+
+export { getCtx, runInCtx }
```

Comment:
Why reexport?

Reply:
Just thought this is part of the `core` domain.

Reply:
Will it be zx api? Or used somewhere else?

Reply:
"./core": "./src/core.mjs",

Reply:
In future, it may become plugin API or smth like that.

Reply:
As it may change. Let's not export in for now.

Reply:
Move to experimental maybe?

Reply:
Yes, we can reexport there.

Reply:
Done.